api/HTMLImageElement.json

```
@@ -831,7 +831,7 @@
           "status": {
             "experimental": false,
             "standard_track": true,
-            "deprecated": true
+            "deprecated": false
```

This is specified as non-conforming: https://html.spec.whatwg.org/multipage/obsolete.html#attr-img-name

```suggestion
            "deprecated": true
```